frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -79,39 +81,43 @@ export default function SideNavigation({
   page,
   currentUser,
   toggleDrawer,
-  onError,
   onResetErrorMessage,
   drawerOpen = false,
   pageTopStyling = {},
 }: SideNavigationProps) {
   const [reviewCount, setReviewCount] = useState(0)
   const theme = useTheme()
+  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
+
+  const doesNotContainUserResponse = useCallback(
```

Comment:
This var name and the logic below doesn't match. The logic is checking that the user hasn't approved the review, but the var name suggests it should be looking to see if the user has responded at all.

I'm guessing the logic is correct and the name should probs be changed to something like `doesNotContainUserApproval`. Otherwise the logic will need updating to remove the `response.decision === Decision.Approve` check.